core/data/fetch.py

```diff
@@ -125,6 +130,58 @@ def img_path_from_row(row: pd.Series, index: int, column="identifier", extra=Non
         if extra:
             out += extra
         return out + "_0.npy"
+    @staticmethod
+    def pad_dataset(df, raw_dataset_path: str, raw_label_path: str, min_amount_of_pictures=3):
+        run_correction = False
+
+        print("coo coo", df)
+
+        values = df['species'].value_counts().keys().tolist()
+        counts = df['species'].value_counts().tolist()
+
+        itt = 0
+
+        less_than_list = []
+
+        for count in counts:
+            if min_amount_of_pictures > count:
+                less_than_list.append((values[itt], count))
+                run_correction = True
+            itt += 1
```

Comment:
Evt. brug `for itt in range(len(counts))` i stedet for at have variabel optælling :)